internal/tests/e2e_sc_happy_path.go

```diff
@@ -0,0 +1,431 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	scTypes "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	scClient "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	bucTypes "github.com/kyma-project/kyma/components/binding-usage-controller/pkg/apis/servicecatalog/v1alpha1"
+	bucClient "github.com/kyma-project/kyma/components/binding-usage-controller/pkg/client/clientset/versioned"
+
+	"github.com/pkg/errors"
+	appsTypes "k8s.io/api/apps/v1beta1"
+	k8sCoreTypes "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+)
+
+const timeoutPerStep = time.Minute
+
+// E2EServiceCatalogHappyPathTest tests the Service Catalog basic functionality:
+// - Creating ServiceInstance
+// - Create ServiceBininding
+// - and injecting those bindings to sample application
+type E2EServiceCatalogHappyPathTest struct {
```

Comment:
please add prerequisites, for example that we have SC for redis.

Reply:
done